testutil/dbconfig/config.go

```diff
@@ -86,7 +85,7 @@ func scanQueries(data []byte, atEOF bool) (advance int, token []byte, err error)
 	if atEOF && len(data) == 0 {
 		return 0, nil, bufio.ErrFinalToken
 	}
-	if i := bytes.LastIndex(data, []byte(";\n")); i >= 0 {
+	if i := bytes.Index(data, []byte(";\n")); i >= 0 {
```

Comment:
Would you add a test for `scanQueries` with `data` that has a few queries?  Just to catch future breakage. Thanks.